css/style.css

```
@@ -1,19 +1,96 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
-}
-
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
-
-
+    /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+    
+    body {
+        font-family: 'Roboto', sans-serif;
+        -webkit-font-smoothing: antialiased;
+    }
+    
+    ul {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-around;
+        margin-left: 550px;
+        margin-right: 150px;
+    }
```

```suggestion
    .header {
        display: flex;
        justify-content: space-between;
        padding: 1rem;
    }
    .header img {
        height: 2rem;
        width: 2rem;
    }
    ul {
        display: flex;
        list-style: none;
        margin: 0;
    }
    .header li {
        padding: 1rem;
    }
```
Instead of using a specific `margin-left` to create space between the logo and the links, we can do this instead.

- We can use flex to put space between the logo and the links
- We can add padding to the list links so that there's space between them